forte/processors/base/data_augment_processor.py

```diff
@@ -40,11 +44,15 @@ def cast(self, pack: DataPack) -> MultiPack:
 
         """
         p = MultiPack()
-        p.add_pack_(pack, self.configs.pack_name)
+        p.add_pack_(pack, "original")
+        augmented_data: Iterable[Tuple[str, DataPack]] = self.augment(pack)
+        for pack_name, data_pack in augmented_data:
+            p.add_pack_(data_pack, pack_name)
         return p
 
-    @classmethod
-    def default_configs(cls):
-        return {
-            'pack_name': 'default'
-        }
+    @abstractmethod
+    def augment(self, pack: DataPack) -> Iterable[Tuple[str, DataPack]]:
```

Comment:
The docstring here only says the programming behavior. But didn't explain it conceptually. Why it returns tuples of pack name and pack? What are the packs? What are the strings? 

You also haven't mentioned that this is an `Iterator`.